opta/flyte.yaml

```diff
@@ -0,0 +1,162 @@
+environments:
+  - name: default
+    path: "./env.yaml" # NOTE: relative path to environment
+    variables:
+      region: {region}
```

Comment:
Can we use `<>` for this maybe? It's a bit confusing what needs to be filled in by the user here and what will be interpolated by opta.